packages/e2e-test-utils/src/inserter-toggle-has-focus.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Asserts that a content block's inserter toggle has keyboard focus
+ * @returns {Promise} A promise that's resolved when the active element is evaluated and asserted to have the inserter toggle's classname.
+ */
+
+export async function inserterToggleHasFocus() {
```

Comment:
This helper is used only in one place, how about we don't expose it as part of public API until it turns out to be useful in other places? For the time being, we can move it to the file where it is used.